pandas/_libs/missing.pyx

```diff
@@ -383,6 +381,29 @@ class NAType(C_NAType):
     __abs__ = _create_unary_propagating_op("__abs__")
     __invert__ = _create_unary_propagating_op("__invert__")
 
+    # pow has special
+    def __pow__(self, other):
+        if other is C_NA:
+            return NA
+        elif isinstance(other, (numbers.Number, np.bool_)):
+            if other == 0:
+                return other
```

Comment:
This should actually be 1 right?
But, the question is then if this should be int or float, and I suppose it should be of the same type as other?

Reply:
Making it the same type as other is probably best.